lib/govuk_index/presenters/details_presenter.rb

```diff
@@ -10,14 +12,24 @@ def indexable_content
       sanitiser.clean(elements)
     end
 
+    def licence_identifier
+      details['licence_identifier']
+    end
+
+    def licence_short_description
+      details['licence_short_description']
+    end
+
   private
 
     attr_reader :details, :sanitiser
 
     def elements
-      details.flat_map do |key, value|
+      content = details.flat_map do |key, value|
+        next unless ALLOWED_FIELDS.include?(key)
```

Comment:
not totally happy with combination of guard clause `if` and inline `if` but couldn't find a nicer way to write it.